staging/prometheus-operator/patch/patch_6_mesosphere_values.sh

```diff
@@ -7,13 +7,15 @@ source $(dirname "$0")/helpers.sh
 
 set -x
 
-cat << EOF >> "${BASEDIR}"/values.yaml
+SRCFILE="${BASEDIR}"/values.yaml
 
+sed -i '' -e '/# Create mesosphere specific resources/,$d' ${SRCFILE}
+
+cat << EOF >> ${SRCFILE}
 # Create mesosphere specific resources
 mesosphereResources:
   create: false
   rules:
-    etcd: true
     velero: false
```

Comment:
Why do we remove etcd from here?

Reply:
Oops sorry I forgot to make a comment on that - we added etcd rules in the past since they weren’t included upstream, but I noticed that upstream does have etcd rules now so no need for us to add them as well